### DISCUSSION TITLE
No clear requirements for black and white color on onboarding

### DISCUSSION THREAD

**Discussion by churik** [status-im/status-legacy#16676] [General]
# Bug Report
> - Design team have not given us a clear explanation of how this is to be handled for black and white yet so we are unable to build it correctly. IMO it is probably better that we remove this option, will this discuss with design team but for now it will not be working correctly as long as it is selectable.
(context https://github.com/status-im/status-mobile/pull/15401#issuecomment-1479492006)
## Problem 
It is a follow-up issue on https://github.com/status-im/status-mobile/pull/15401
Causes issues when black and white color is selected

#### Expected behavior
All cases are defined in the design

#### Actual behavior

![IMAGE 2023-03-22 12:43:00](https://user-images.githubusercontent.com/4557972/226893898-c3c44106-d358-4bf5-ae2c-b3e968123369.jpg)

![IMAGE 2023-03-22 12:42:56](https://user-images.githubusercontent.com/4557972/226893885-4d51abfe-3e00-40a7-93c8-2a50da18c1cf.jpg)

**OS:** IOS, Android

**Comment by churik** [comment]
cc @xAlisher @pedro-et

**Comment by J-Son89** [comment]
@xAlisher @mariocnovoa @pedro-et can we get many examples of how black and white custom color should work, for both dark and light theme. and many different components /pages

**Comment by J-Son89** [comment]
also cc: @Parveshdhull as he is quite knowledgeable on this area :)

**Comment by J-Son89** [comment]
some examples - https://www.figma.com/file/PiN0uwRgZj4L9U4N7xO7Xb/Profile-for-Mobile?type=design&node-id=5973%3A247601&t=UD0ood9H6e7KybNt-1
